src/main/java/mate/jdbc/service/impl/CarServiceImpl.java

```diff
@@ -0,0 +1,67 @@
+package mate.jdbc.service.impl;
+
+import java.util.List;
+import mate.jdbc.dao.CarDao;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.lib.Service;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.service.CarService;
+
+@Service
+public class CarServiceImpl implements CarService {
+    @Inject
+    private CarDao carDao;
+
+    @Override
+    public Car create(Car car) {
+        return carDao.create(car);
+    }
+
+    @Override
+    public Car get(Long id) {
+        return carDao.get(id)
+                .orElseThrow(() ->
+                        new DataProcessingException("Can't get car by id: " + id));
+    }
+
+    @Override
+    public List<Car> getAll() {
+        return carDao.getAll();
+    }
+
+    @Override
+    public Car update(Car car) {
+        return carDao.update(car);
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        return carDao.delete(id);
+    }
+
+    @Override
+    public void addDriverToCar(Driver driver, Car car) {
+        try {
+            if (!car.getDrivers().contains(driver)) {
+                car.getDrivers().add(driver);
+                carDao.update(car);
+            }
+        } catch (DataProcessingException e) {
+            throw new DataProcessingException("Error occurred while "
+                    + "adding driver to car", e);
+        }

```

Comment:
```suggestion
                car.getDrivers().add(driver);
                carDao.update(car);
```
As agreed with other mentors, we'd recommend removing this check and not throwing any exceptions here.